app/controllers/account/profile_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Account::ProfileController < ApplicationController
+  before_action :authenticate_user!
+
+  def edit
+    @profile = current_user
+  end
+
+  def update
+    @profile = current_user
+    if @profile.update_attributes(user_params)
+      flash[:success] = "Profile updated!"
+      redirect_to account_user_path(@profile)
```

Comment:
wrong path.
should be account_profile_path
(without @profile)
since account_user_path is only for admins